ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/clientlibs/site/js/tabs.js

```diff
@@ -87,7 +87,9 @@
             for (var i = 0; i < hooks.length; i++) {
                 var hook = hooks[i];
                 if (hook.closest("[data-cmp-is=" + Tabs.IS + "]") === this._elements.self) { // only process own tab elements
-                    var key = hook.dataset[Tabs.NS + "Hook" + "Adaptiveformtabs"];
+                    var lowerCased = Tabs.IS.toLowerCase();
+                    var capitalized = lowerCased.charAt(0).toUpperCase() + lowerCased.slice(1);
+                    var key = hook.dataset[Tabs.NS + "Hook" + capitalized];
```

Comment:
we already have an API Utils.readData for this. can you pls try to reuse it?

Reply:
@nit23uec ,
I had checked that api, it provides an array of key-value pair of dataset key and value for any html element passed. SO to attain the value from that object for the particulat key, we will have to write this logic anyway here